course_content/notebooks/numpy_intro.ipynb

```diff
@@ -995,7 +995,7 @@
      "cell_type": "code",
      "collapsed": false,
      "input": [
-      "arr = np.arange(8).reshape(2, 4)\n",
+      "arr = np.arange(2, 18, 2).reshape(2, 4)\n",
```

Comment:
How about using linspace. I'm not fond of making an array of an expected length without specifying the length itself.